docs/resources/application.md

```diff
@@ -29,7 +29,7 @@ resource "juju_application" "this" {
 
   placement = "0,1,2"
 
-  storage = {
+  storage_directives = {
```

Comment:
Just checked the schema and it seems there are two fields, storage and storage_directives. Should we have both instead of removing this?

Reply:
Nope, one is to "ask for the storage". Another is to "manage the state of the storage in TF".

Reply:
Coolcool